src/lib/utils/directives.tsx

```diff
@@ -0,0 +1,9 @@
+export const ifDirective = (c: any, nIf: boolean) => {
+  return nIf || typeof nIf === 'undefined' ? c : null
+}
+
+export const renderWithDirectives = (element, { nIf }: { nIf?: boolean }) => {
```

Comment:
Is this function written in this way to prepare for future directives? Otherwise this would be a bit neater:

```
return ifDirective(element, nIf)
```

Reply:
Click will be added shortly!